### ISSUE TITLE
Latest builds 1.40.0 and forward have broken intellisense, autocomplete, etc.

### ISSUE THREAD

**Issue by KBassford** [VSCodium/vscodium#289]
**Describe the bug**
Symptoms are as follows ...
- intellisense only displays "loading ...", not definitions.
- "jump to" does not work.
- autocomplete not working.
- auto insertion of modules not  working.
- debug on save not working.

**Please confirm that this problem is VSCodium-specific**
- Unknown, stopped using VSCode as the method used for external URL's broke my machine.
(not present in last version of vscode I used (1.35.0))

**Please confirm that the issue/resolution isn't already documented**
- checked here and vscode repo and my exact issue on my OS is not mentioned in either location.

**To Reproduce**
1. Open a Go project

**Expected behavior**
All the problems noted in "symptoms" should not be present.

**Screenshots**
![Screenshot at 2019-11-20 13-33-46](https://user-images.githubusercontent.com/31860128/69269131-62e2f200-0b9e-11ea-9df0-fb9c6cd0ed41.png)
(Note: I just saved the file, in the highlighted line, "f" is not defined and would normally appear in the nearby "Problems" display.)

**Desktop (please complete the following information):**
 - OS: Parrot GNU/Linux 4.7
 - Architecture x64
 - Version 4.7
codium 1.40.1 with Go plugin 0.11.9
go version go1.13.4 linux/amd64

**Additional context**
Just started today, almost impossible to work as codium is about a useless as notepad now.

**Comment by stripedpajamas** [comment]
I thought I experienced something like this as well yesterday but when I closed and reopened it was working. 

Is anything helpful logged in the Output pane at the bottom under Main or Extensions?

**Comment by KBassford** [comment]
Not really.  I did have some messages about "gopkg" (I may be off by a letter) being out-of-date, but it claims to have updated fine.  My installation updated first to 1.40.0 (which included a note that the release would break themes and such), so I switched to a couple of different themes, same result.  Then I found version 1.40.1 and forced an update to that, same results.  I'm about ready to uninstall and reinstall as codium is pretty useless right now, might as well be using vi.

**Comment by stripedpajamas** [comment]
Can you outline how you "forced" the update to 1.40.1?

**Comment by KBassford** [comment]
I had two repositories ...
```
apt-cache showpkg codium
Package: codium
Versions: 
1.40.1-1573814789 (/var/lib/apt/lists/gitlab.com_paulcarroty_vscodium-deb-rpm-repo_raw_repos_debs_dists_vscodium_main_binary-amd64_Packages) (/var/lib/dpkg/status)
 Description Language: 
                 File: /var/lib/apt/lists/gitlab.com_paulcarroty_vscodium-deb-rpm-repo_raw_repos_debs_dists_vscodium_main_binary-amd64_Packages
                  MD5: b38f8538d3cead7a194af6aeeaf02e6a
 Description Language: 
                 File: /var/lib/apt/lists/deb.parrot.sh_parrot_dists_rolling_main_binary-amd64_Packages
                  MD5: b38f8538d3cead7a194af6aeeaf02e6a

1.40.0-1573156533 (/var/lib/apt/lists/deb.parrot.sh_parrot_dists_rolling_main_binary-amd64_Packages) (/var/lib/apt/lists/mirrors.mit.edu_parrot_dists_rolling_main_binary-amd64_Packages) (/var/lib/apt/lists/mirror.clarkson.edu_parrot_dists_rolling_main_binary-amd64_Packages)
 Description Language: 
                 File: /var/lib/apt/lists/gitlab.com_paulcarroty_vscodium-deb-rpm-repo_raw_repos_debs_dists_vscodium_main_binary-amd64_Packages
                  MD5: b38f8538d3cead7a194af6aeeaf02e6a
 Description Language: 
                 File: /var/lib/apt/lists/deb.parrot.sh_parrot_dists_rolling_main_binary-amd64_Packages
                  MD5: b38f8538d3cead7a194af6aeeaf02e6a

```
For reasons only the developers of Parrot OS know, "apt install -u  codium" would default to 1.40.0, not the later version.  So I used the command below to update the version ...
```
sudo apt install codium=1.40.1-1573814789
```

**Comment by KBassford** [comment]
I also found this ...
https://forum.manjaro.org/t/visual-studio-code-broken-in-unstable/110700
so it might be upstream.  I wish someone had codium 1.39.2 available so I could downgrade.  Both servers listed above don't have a copy as far as I can tell.

**Comment by KBassford** [comment]
Well I'm beside myself.  I tried ...
1. Downgrading to 1.39.2, no effect.
2. Completely uninstalling, purge, then installing 1.40.1 (in the same way as above).  Still no effect.
Looking to see if maybe firejail is doing something to block codium or Go.

**Comment by KBassford** [comment]
I think I found the issue!  I began centering on "gopls" as everything that was malfunctioning seems to funnel through this module.  I found that renaming go.mod to go.oldmod has restored functionality.  A mismatch between modules actually in use, and those I recently eliminated from my code appears to be the cause.

**This will apparently take out ANY editor, not just codium!**

This fix should be added to a troubleshooting page somewhere so folks like me don't waste your time or that of others.  I am so glad I found this.  Now I just have to regenerate my go.mod such that it uses only dependencies that currently exist.  (I recently revised my code to eliminate several dependencies, based on the philosophy that "less is better".)

I am closing this issue, with the hope that this critical information is posted somewhere where users can find it.  Learn something everyday.  Thank you for your time.
